src/frontends/pytorch/src/op/expm1.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/exp.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_expm1(const NodeContext& context) {
+    num_inputs_check(context, 1, 2);
+    // aten::expm1(Tensor self) -> Tensor
+    // aten::expm1(Tensor self, Tensor out) -> out Tensor
+    auto input = context.get_input(0);
+    auto input_dtype = input.get_element_type();
+
+    if (input_dtype.is_dynamic() || !input_dtype.is_real()) {
+        input = context.mark_node(std::make_shared<v0::Convert>(input, element::f32));
+    }
+
+    auto exp = context.mark_node(std::make_shared<v0::Exp>(input));
```

Comment:
```suggestion
    auto exp = op::translate_1to1_match_1_inputs_with_fp32_type_alignment<v0::Exp>(context);
```
Use the same code that is used to convert `aten::exp` to create `exp` and what is left is to do `-1` for it.

Reply:
Yes, you are right, you need to take first element from vector

Reply:
> Please also check with `PYTORCH_OP_CONVERSION_CHECK` that vector has 1 element

Sure, thanks!